GLS edit of Tech Exercise ii (CI/CD) completed. (#102)
rht-labs/tech-exercise: docs/slides/content/tech-exercise-ii.md

#### Feedback
* Q & A
* Real World Stories
* Pull Requests Welcome 🦄

#### WHO - Pipelines <!-- .element: class="title-bottom-left" -->
<!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->

> <!--Tech Edit - this slide is empty except for the heading. -->

Reviewer: @springdo do you have any screenshots you can upload? I also can dig a bit deeper and bring out something :D

#### WHO - App of Apps <!-- .element: class="title-bottom-left" -->
<!-- .slide: data-background-size="contain" data-background-image="images/tech-exercise-ii/example-who.png", class="black-style" data-background-opacity="1"	 -->

> <!--Tech Edit - this slide is empty except for the heading. -->

Reviewer: @springdosame goes for this one :D

### Summary
* **Continuous Integration**: Integrate several changes into a shared repository and rely heavily on testing to make sure that the changes and commits work and have passed all tests (unit testing / code quality / code conduct).
* **Continuous Delivery**: The goal is to have a working artifact that's ready to be deployed at any time. It relies on testing and automation to make sure that it has passed all functional tests and can be manually deployed (push button).

> * **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, is to rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.

Reviewer: ```suggestion
* **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, we can now rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

Author: The grammar is not quite right in the suggestion. If we leave out the clause set off by commas, we have "A further step, ..., now we can..." which isn't correct.

> * **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, is to rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.

Author: ```suggestion
* **Continuous Deployment/Release**: After we are sure that this artifact is ready to deploy to production, we can take a further step, relying on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

> * **Continuous Deployment/Release**: A further step, now that we are sure this artifact is ready to deploy to production, is to rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.

Author: ```suggestion
* **Continuous Deployment/Release**: Now that we are sure this artifact is ready to deploy to production, we can take a step further and rely on features toggle, dark launches, or other deployment strategies to promote this artifact to other critical environments. By using these techniques for the new features/fixes, we can release them immediately without risking the operation.
```

> <!--TechEdit: This last bullet didn't make sense as written, it was sort of a long run-on. I've rewritten, please check. -->

Reviewer: ```suggestion
```
